jest.config.js

```diff
@@ -1,5 +1,5 @@
 const { pathsToModuleNameMapper } = require('ts-jest/utils');
-const { compilerOptions } = require('./tsconfig');
+const { compilerOptions } = require('./tsconfig.test');
```

Comment:
Why `tsconfig.test` here?

Reply:
How about `tsconfig.jest`?

Reply:
`tsconfig.jest` looks great 👍 , but I have to revise the workflow to fix the CI problem.
I'll close this pull request, and repone it again after I fix the CI problem.